docker/Dockerfile

```diff
@@ -27,7 +27,7 @@ WORKDIR $GOPATH/src/github.com/go-debos/debos/cmd/debos
 RUN go install ./...
```

Comment:
need to make the first stage bookworm too - otherwise we're building on one distro and running on another ;-)